src/app/plugins/pubsub.plugin.ts

```diff
@@ -28,60 +32,51 @@ export class PubSubPlugin extends Plugin {
         const subs = response.data;
 
         this._subs = subs.map((subData: { name: string }) => {
-          return subData.name
-            .toLowerCase()
-            .split('-')
-            .join(' ');
+          return subData.name.toLowerCase();
```

Comment:
if you omit the {} in the lambda: `=> subData.name.toLowerCase()` , no need to say `return`